### DISCUSSION TITLE
1.0.9

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsAppPatcher#26] [Announcements]
Fix signature padding and add some files to .gitignore

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsAppPatcher/releases/tag/1.0.9'>1.0.9</a>.</em>

**Comment by naive890** [comment]
so this version of WhatsApp has now become out of date, just want to know if you have plans to release an update for the same that would be really helpful, really appreciate you. thankyou.

**Reply by jeverling** [reply]
Preferably a non-beta version so it doesn't expire 🙂

**Reply by naive890** [reply]
i came across this mod just 3 days ago
 .... you've tried it? does the view once media saving works?

**Reply by jeverling** [reply]
Sorry I haven't tested that, was only interested in keeping deleted messages. So I can't say. But I suppose it does, haven't seen any open issues relating to that.

**Reply by naive890** [reply]
is there  like a way of contacting him other than this because i really cant find any other trustable mods who provide these features... i really hope they update this

**Reply by Schwartzblat** [reply]
Hi, I will fix it at the end of the next week.

**Reply by naive890** [reply]
thankyou really appreciate it. will try it out once you've updated it.

**Reply by coconutbro** [reply]
Bro pls make the feature where we can Edit sended msg without knowing anyone in WhatsApp group if we edit the It shows  ( Edited massage) both side

**Reply by Schwartzblat** [reply]
Let me know if it was fixed @naive890 @jeverling .
@coconutbro I am not sure it is possible.

**Reply by coconutbro** [reply]
Pls try to make it if it is possible.

**Reply by naive890** [reply]
i will try it out tomorrow because ive to take all the backup first....ill let u know

**Reply by coconutbro** [reply]
What happened??

**Comment by coconutbro** [comment]
![Screenshot_2023-08-24-00-04-33-24_6012fa4d4ddec268fc5c7112cbb265e7](https://github.com/Schwartzblat/WhatsAppPatcher/assets/143184084/8f1667e3-1b14-4408-99c8-fe01322c2da1)

**Comment by jeverling** [comment]
> Let me know if it was fixed @naive890 @jeverling . @coconutbro I am not sure it is possible.

The latest version crashes for me. I have added some details in https://github.com/Schwartzblat/WhatsAppPatcher/issues/29

**Comment by naive890** [comment]
> Fix signature padding and add some files to .gitignore
> 
> _This discussion was created from the release [1.0.9](https://github.com/Schwartzblat/WhatsAppPatcher/releases/tag/1.0.9)._

so ive tried this new version on multiple devices yet as soon as the WhatsApp meta screen appears the app crashes. this wasnt a issue for the previous versions. can u still fix this?
